Gruntfile.js

```diff
@@ -472,7 +472,9 @@ module.exports = function(grunt) {
       },
       'start-webdriver-ci': {
         cmd:
-          'scripts/e2e/start_webdriver.sh'
```

Comment:
Are we going to remove the `start_webdriver.sh` script? or are we keeping it because it's used/useful in other scenarios?